### ISSUE TITLE
[BUG] Loading Simbrief routes and STARs with actual AIRAC circle are "Not in Database"

### ISSUE THREAD

**Issue by FGrabi** [flybywiresim/a32nx#3403]
**Mod Version**
{
  "built": "2021-02-07T05:52:09+00:00",
  "ref": "master",
  "sha": "0f2481ad9161e49a97ed3a379a5f8c3293e42461",
  "actor": "tyler58546",
  "event_name": "manual"
}

**Describe the bug**

Created in SB a flight with the actual AIRAC circle 2101 from EINN->ESSD

**AGIN3A AGINI DCT ERNAN DCT MAC UN537 GOW L983 SAB UM89 RIVOT UM82 NIVUN P609 KSD DCT XOTBU XOTB1F**

When loading the flight in the MCDU INIT1 Page (after _Init Data Request"_ in MCDU ATSU), the message AWY WYP Not in Database was shown. Add in the FP the Rwy's and SID/STAR, checking the FP, the route was _shorten_ to **AGIN3A AGINI DCT ERNAN DCT XOTBU XOTB1F _(but without the WP of the STAR)_ and the ILS Z 32 WP**. Add the WYP MAC manual as _Next Waypoint_ for the ERNAN successfuly and try to add from MAC the AW _UN357_ with no success. So I added as Next Waypoint all the "Endpoints" of the missing Airways (GOW, SAB, RIVOT, NIVUN, KSD). On the Route Display it looks fine, so I start the flight.
During the flight I found, that the "Distance" at the FP-Display of the FMGC was totally different from the ND (see screenshots) and that the total distance to the Airport was "frozen" - no decrease. (see screenshots).
The difference was logical for me for the NIVUN P609 KSD Airway, because I did not entered in the beginning all the waypoints on the P609 (what I did during the flight, after I recognized it on the LiveMap of STKP (SimToolKitPro) and NAVIGRAPH CHARTS) But the screeshot shows the situation between the WP RIVOT UM82 NIVUN which has **NO** additional WPs between RIVOT and NIVUN.
After I passed the XOTBU WP - which is the beginning of the XOTB1F STAR, the plane goes direct to ESSD. I changed this with DIR-TO to the next known WP on the ILS Approach and was luckely able to land the Plane correct, because this behavior is really anoying, especially if you fly Online with IVAO and Real ATC.

Also in the flight from LPAZ-EINN before the mentioned one, with the following SB Route

XUVA1N XUVAP DCT BAVAS DCT **42N022W** DCT **45N020W** DCT KOGAD DCT ATSUR DCT BUNON BUNO2E

both coordinated waypoints are ignored and also reported as "Not in Database". I don't know which MOD Version I used for this flight, I alwas load the latest Dev-Version just before starting the MSFS. The flight starts: 2021-02-06 14:35:48 (if this hepls)

**To Reproduce**

1. USE in SB the actual AIRAC 2101
2. Load via Navigraph Installer for MSFS the actual AIRAC 2101 into the MSFS Community Folder
3. Start Cold/Dark at EINN
4. Load via MCDU the actual SB Flight (OFP)
3. Watch the show ;-)

**Expected behavior**

That the actual AIRAC circle is known and **USED**

**Actual behavior**

see above

**References**

See the difference with the distance

![Microsoft Flight Simulator Screenshot 2021 02 07 - 16 45 48 23](https://user-images.githubusercontent.com/74778044/107156686-155bcc00-6980-11eb-96f2-74bce9bb99b4.png)
![Microsoft Flight Simulator Screenshot 2021 02 07 - 16 45 59 74](https://user-images.githubusercontent.com/74778044/107156689-1b51ad00-6980-11eb-8a5c-bd0ad9a27c84.png)

**Additional context**

Discussed with @pareil6 in Issue #3385

Hope all is understandable, sorry I'm not a native english speaker 

Was this working before/when did the issue start occurring?
Had sometimes Problems with the STARs not flown correct and was wondering that in MCDU the loaded AIRAC was not shown up See #3233 
Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):

**Comment by pareil6** [comment]
Identified issue in SID detection - if the SID only has one waypoint (the exit fix), it'll incorrectly try to add the waypoint as if it's on an airway, using the SID itself as the airway name, causing the AWY/WPT MISMATCH warning and causing a cascade failure afterwards since the first waypoint isn't inserted (so the next airway is invalid, etc).

I've got a fix locally, will roll into a PR with a couple of other SimBrief issues I've looked at today.

**Comment by MisterChocker** [comment]
Duplicate of #2923.
